### DISCUSSION TITLE
Carla docker for 0.9.14?

### DISCUSSION THREAD

**Discussion by chrisochoatri** [carla-simulator/carla#6071] [Q&A]
Hello, the 0.9.14 update looks really interesting, thank you for all the hard work! I'm curious if there are plans to update the carlasim/carla:latest image with 0.9.14 or is official docker support only for 0.9.13 and earlier?

**Comment by Kin-Zhang** [comment]
Update on their page now: 
```bash
docker pull carlasim/carla:0.9.14
```
https://hub.docker.com/r/carlasim/carla/tags
